tests/tsan-flow.cc

```diff
@@ -0,0 +1,64 @@
+#include <chrono>
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include "sentinel-core/flow/flow_rule_manager.h"
+#include "sentinel-core/init/init_target_registry.h"
+#include "sentinel-core/log/log_base.h"
+#include "sentinel-core/log/logger.h"
+#include "sentinel-core/log/metric/metric_log_task.h"
+#include "sentinel-core/public/sph_u.h"
+#include "sentinel-core/transport/command/http_server_init_target.h"
+
+std::atomic<int> g_count{0};
+
+Sentinel::Init::InitTargetRegister<
```

Comment:
Duplicate initialization of `HttpCommandCenterInitTarget`?

Reply:
Sorry, it is indeed redundant, I will delete it.